build_ssl.sh

```diff
@@ -99,6 +99,7 @@ build_ssl_1_1() {
     cp libcrypto.a libssl.a "../$version_out_dir/$qt_arch" || exit 1
     ln -s "../$version_out_dir/$qt_arch/libcrypto_1_1.so" "../$version_out_dir/$qt_arch/libcrypto.so"
     ln -s "../$version_out_dir/$qt_arch/libssl_1_1.so" "../$version_out_dir/$qt_arch/libssl.so"
+    ln -s "../$version_out_dir/include" "../$version_out_dir/$qt_arch/include"
```

Comment:
why include needs to be added to each abi dir?

Reply:
So it works out if the boxwith the findopenssl module of CMake. 

Like this the `OPENSSL_ROOT_DIR` can be set to the ABI folder and CMake will automatically resolve the include folder when using `find_package(OpenSSL)`

Reply:
Okay, that makes sense, thanks!